src/main/java/dukeApp/files/DateTime.java

```diff
@@ -0,0 +1,32 @@
+package dukeApp.files;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class DateTime {
```

Comment:
I like how you have a class specifically to hold date and time and their functions